### PR TITLE
Added missing float32 privative type support

### PR DESCRIPTION
**Pull request by goldfish1974**
I noticed that the float32 privative was missing.

Added this and matching tests for the next release.

### CODE REVIEW

File: src/FSharp.UMX.fs [hunk 1]
```diff
@@ -33,6 +33,7 @@ type UMX =
     static member inline tag<[<Measure>]'m> (x : int64) : int64<'m> = Unsafe.cast x
     static member inline tag<[<Measure>]'m> (x : uint64) : uint64<'m> = Unsafe.cast x
     static member inline tag<[<Measure>]'m> (x : float) : float<'m> = Unsafe.cast x
+    static member inline tag<[<Measure>]'m> (x : float32) : float32<'m> = Unsafe.cast x
```
Comment (eiriktsarpalis) [inline src/FSharp.UMX.fs:36]: Minor nit-pick: order lower precision types above higher precision types.
```suggestion
    static member inline tag<[<Measure>]'m> (x : float32) : float32<'m> = Unsafe.cast x
    static member inline tag<[<Measure>]'m> (x : float) : float<'m> = Unsafe.cast x
```

File: tests/FSharp.UMX.Tests.fs [hunk 2]
```diff
@@ -33,6 +34,7 @@ let ``Simple unit of measure conversions with cast operator``() =
     let w : string<foo> = % sprintf "%O %s %d" %x %y %z
     let b : byte<foo> = % 1uy
     let s : int16<foo> = % 1s
+    let f : float32<foo>  = % 10.0f
```
Comment (eiriktsarpalis) [inline tests/FSharp.UMX.Tests.fs:37]: Might make sense to add something that validates other common aliases work as well
```suggestion
    let f : float32<foo>  = % 10.0f
    let f : single<foo> = % 10.0f
```

File: src/FSharp.UMX.fs (hunk 1 above)
Comment (goldfish1974) [reply src/FSharp.UMX.fs:36]: I didn't even think about that. Thank for the nit-pick.

**Comment by eiriktsarpalis** [comment]
Thanks!

**Pull request merged**